src/engine/tt_hash_table.h

```diff
@@ -14,20 +15,87 @@ enum TtHashFlag : uint8_t {
     TtHashBeta = 2,
 };
 
+/*
+ * Non-key members are encoded for compression. From lower bits to higher, entries are:
+ * depth: 7 bits, i.e. 0000 ... 0000 0000 0111 1111
+ * flag: 3 bits, i.e. 0000 ... 0000 0011 1000 0000
+ * score: 17 bits (see board_defs.h)
+ * generation: 8 bits
+ * move: 26 bits (see move_types.h)
+ * */
+
+class TtHashEntryData {
+public:
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wshift-count-overflow"
```

Comment:
With my comments you should probably on longer see this warning

Reply:
That's right. I've removed the ignore.